Example/RZTouchIDDemo/RZAppDelegate.h

```diff
@@ -7,11 +7,13 @@
 //
 
 @import UIKit;
-
+@class RZTouchID;
 @interface RZAppDelegate : UIResponder <UIApplicationDelegate>
 
 @property (strong, nonatomic) UIWindow *window;
+@property (strong, nonatomic) RZTouchID *touchIDService;
```

Comment:
I see there is a property and also a shared instance. I'm assuming the property is obsolete?